src/Domain/Creators/CylindricalBinaryCompactObject.cpp

```diff
@@ -417,8 +417,8 @@ CylindricalBinaryCompactObject::CylindricalBinaryCompactObject(
         std::array{rotate_from_z_to_x_axis(center_A_),
                    rotate_from_z_to_x_axis(center_B_)},
         std::nullopt, std::nullopt, std::array{radius_A_, outer_radius_A_},
-        std::array{radius_B_, outer_radius_B_}, inner_common_radius,
-        outer_radius_);
+        std::array{radius_B_, outer_radius_B_}, include_inner_sphere_A,
+        include_inner_sphere_B, inner_common_radius, outer_radius_);
```

Comment:
I think `include_inner_sphere_{A/B}` should be replaced with just `False` here, right? We don't allow central object cubes in the cylindrical domain. This inner sphere is just another set of wedges, but isn't filled